c/src/CMakeLists.txt

```diff
@@ -31,10 +31,27 @@ set(libsbp_SRCS
   edc.c
   sbp.c
   )
-set(LIBSBP_CFLAGS "-fpic" CACHE STRING "Compile flags for libsbp.")
+
+set(LIBSBP_CFLAGS "-fpic -Wall -Werror" CACHE STRING "Compile flags for libsbp.")
 
 add_library(sbp ${libsbp_SRCS})
-target_compile_options(sbp PRIVATE "${LIBSBP_CFLAGS}")
+
+if(MSVC)
+    separate_arguments(LIBSBP_COMPILER_OPTIONS WINDOWS_COMMAND ${LIBSBP_CFLAGS})
```

Comment:
This doesn't compile with MSVC:
`cl : Command line error D8021: invalid numeric argument '/Werror'`

Please check https://github.com/swift-nav/libsbp/pull/647

Reply:
Good catch! I'll add a special case for MSVC. I think the intention is for the consumers of the library that need/want specific options to set this string themselves, but it would be good to make sure the defaults work out of the box for the common compilers at least :)